viewing_party/party.py

```diff
@@ -1,23 +1,225 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from ast import If
+from types import NoneType

```

Comment:
I don't think you need these imports. Please remove these floaty, random bits in future PRs.